api/README.md

```diff
@@ -1,30 +0,0 @@
-
-# OSF API
-
-## Getting started
-
-From the root osf directory:
-
-```bash
-pip install -r requirements.txt
-# Required for browse-able API
-python manage.py collectstatic
```

Comment:
Is this still required for the web-browsable API tool? This requirement seems to have been dropped in the revised docs.